shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1101,6 +1136,13 @@ boolean isFocusable() {
                     || (hint != null && !hint.isEmpty());
         }
 
+        // We exclude the hint from the node's label if there is a supported hint text
+        // override provided on one of the supported standard actions.
+        boolean hasHintOverride() {
```

Comment:
This seems surprising. What if the hint was about something else?

Reply:
I wasn't sure what the correct answer is here. I don't want to lead people into repeating themselves in the onTapHint.  Happy to remove this logic

Reply:
Removed